src/browser-index.ts

```diff
@@ -16,9 +16,23 @@ limitations under the License.
 
 import * as matrixcs from "./matrix";
 import request from "browser-request";
-import queryString from "qs";
+import queryString, { IStringifyOptions } from "qs";
 
-matrixcs.request(function(opts, fn) {
+interface RequestOptions {
+    qs: string | Record<string | number | symbol, string>;
+    qsStringifyOptions: IStringifyOptions;
+}
+
+enum IndexedDBStoreName {
+    crypto = "matrix-js-sdk:crypto"
+}
+
+type GlobalObject = NodeJS.Global & typeof globalThis & Record<"matrixcs", typeof matrixcs>;
```

Comment:
I thought we already had a type for this? In general I don't think we should be touching `global` and should fix it.